### ISSUE TITLE
Rules CKV_AWS_18 and CKV_AWS_19 fail if s3 resources are defined in a terraform module

### ISSUE THREAD

**Issue by JedGilbert** [bridgecrewio/checkov#3277]
**Describe the issue**

When upgrading the AWS provider in Terraform to a version > 3.75 there has been significant change to the aws_s3_bucket resource. If the S3 resources are referenced in a child module rather than the root level it seems Checkov is still failing CKV_AWS_18 and CKV_AWS_19 based upon our usage. I believe these to be false positives.

CKV_AWS_18: "Ensure the S3 bucket has access logging enabled"

access logging is configured by the resource aws_s3_bucket_logging

CKV_AWS_19: "Ensure all data stored in the S3 bucket is securely encrypted at rest"

encryption at rest is configured by the resource aws_s3_bucket_server_side_encryption_configuration

**Examples**

### modules/s3/main.tf

```

resource "aws_kms_key" "s3_key" {
  description             = "KMS key 1"
  deletion_window_in_days = 10
}

resource "aws_s3_bucket" "bucket" {
  bucket = "sample-bucket"
}

resource "aws_s3_bucket_server_side_encryption_configuration" "bucket" {
  bucket = aws_s3_bucket.bucket.id

  rule {
    apply_server_side_encryption_by_default {
      kms_master_key_id = aws_kms_key.s3_key.key_id
      sse_algorithm     = "aws:kms"
    }
    bucket_key_enabled = false
  }
}

resource "aws_s3_bucket_logging" "bucket" {
  bucket = aws_s3_bucket.bucket.id

  target_bucket = "logging-bucket"
  target_prefix = "sample-bucket/"
}

```
### main.tf

```

module "s3" {
  source = "./modules/s3"
}

```

Command: checkov -f plan.json --check CKV_AWS_18,CKV_AWS_19 --repo-root-for-plan-enrichment "./"

Expected both rules pass for resource aws_s3_bucket.bucket

**Version (please complete the following information):**
 - Checkov Version 2.1.81

**Additional context**
Terraform version: 1.2.6
AWS Provider version: 4.23.0

If I move the contents of the module file to the root module both rules pass as expected.

**Comment by JedGilbert** [comment]
Thanks @gruebel 👍
